.dockerignore

```
@@ -0,0 +1,9 @@
+.env
+.venv
+.git
+.pytest_cache
+./todo_app/__pycache__
```

```suggestion
**/__pycache__
```
The fact you've specified the folder first suggests you are aware that `.dockerignore` has slightly different behaviour to `.gitignore` - in particular in that it _doesn't_ ignore subfolders by default - but here we are probably best using the `**` prefix to mean arbitrary levels of nesting should be considered for this rule since you might have pycache folders under e.g. the `view_models` folder too